packages/lab/src/toast/ToastClose.css

```diff
@@ -0,0 +1,7 @@
+/* Styles applied to ToastClose button */
+.saltButton.saltToastClose {
```

Comment:
I see no visual change when I toggle this class in the dev tools. Like you said in the meeting today, I feel like you could completely remove this component.